### ISSUE TITLE
Changes to Discord's CDN affecting Piped, DDrive, and DDRV

### ISSUE THREAD

**Issue by jacobvd0** [jacobvd0/status.jacob9086.xyz#43]
Discord has just added changes to their CDN that now requires files to have a signature which result in file links expiring after some time. This has been put in place to stop files being accessed outside of Discord.
Before this change links used to look like this: https://cdn.discordapp.com/attachments/channelid/id/file.ext
Now they look like this: https://cdn.discordapp.com/attachments/channelid/id/file.ext?ex=65163669&is=6514e4e9&hm=f24fe35ca20a79fed996247a40c993d080219c518d072c746976cd5802ac49f2&

Once implemented, this change will result in file links expiring after 24 hours (resulting in files no longer working outside of discord after a day).

**TL;DR: Don't upload any important files to Piped, DDrive, or DDRV until this is fixed. Files will start to expire after 24 hours once Discord implements this change.**
